tests/pre.php

```diff
@@ -1,18 +1,27 @@
 <?php
 
 it('renders the element', function () {
-    $html = parse('<pre>
-    <h1>Introduction</h1>
+    $content = '    <h1>Introduction</h1>
 
     <div>The body of your message.</div>
 
     Thanks,
     Laravel
 
-    © 2021 Laravel. All rights reserved.
-</pre>');
+    © 2021 Laravel. All rights reserved.';
+
+    $html = parse("<pre>$content</pre>");
 
-    expect($html)->toBe('
+    expect($html)->toBe(
+        "\n".implode("\n", array_map(
+            fn (string $line) => str_pad($line, 44),
+            explode("\n", $content)
+        ))
+    );
```

Comment:
Can you change the output of this to the output text? Thanks

Reply:
There is a problem to use plain text in this test, cause PHPStorm removes all extra spaces when someone opens this file.

Reply:
Humm, maybe string concat for each line? Thanks

Reply:
What about something like this?

```php
    expect($html)->toBe("
    <h1>Introduction</h1>".str_repeat(' ', 19)."
".str_repeat(' ', 44)."
    <div>The body of your message.</div>".str_repeat(' ', 4)."
".str_repeat(' ', 44)."
    Thanks,".str_repeat(' ', 33)."
    Laravel".str_repeat(' ', 33)."
".str_repeat(' ', 44)."
    © 2021 Laravel. All rights reserved.".str_repeat(' ', 3)
    );
```

Reply:
I think it works.